bin/openstack_taster

```diff
@@ -66,8 +64,11 @@ rescue StandardError => e
   exit(1)
 end
 
+image_name = params[0]
+settings[:ssh_user] = params[1]
+
 require 'fog/openstack'
-require 'openstack_taster'
+require './lib/openstack_taster'
```

Comment:
I added this bit so testing would work locally.

Reply:
Could this just be ``'lib/openstack_taster'`` instead of the leading ``./``?

Reply:
I tried to do that when testing locally, but that didn't work either. The current directory is not in the `$LOAD_PATH` variable.

Reply:
I'm confused. Why does this need to be added in the first place? Have you tried testing it via building the gem and installing it?

Reply:
I was testing this by running `ruby bin/openstack_taster`. I forgot about using the `-I` flag.